ios/Sources/AboutFeature/AboutDroidKaigiScreen.swift

```diff
@@ -110,15 +82,54 @@ public struct AboutDroidKaigiScreen: View {
     }
 }
 
+private extension AboutDroidKaigiModel {
+    var title: String {
+        switch self {
+        case .behaviorCode:
+            return L10n.AboutDroidKaigiScreen.behaviorCode
+        case .opensourceLicense:
+            return L10n.AboutDroidKaigiScreen.opensourceLincense
+        case .privacyPolicy:
+            return L10n.AboutDroidKaigiScreen.privacyPolicy
+        }
+    }
+
+    var image: Image {
+        switch self {
+        case .behaviorCode:
+            return Image(systemName: "book")
+        case .opensourceLicense:
+            return Image(systemName: "star")
+        case .privacyPolicy:
+            return Image(systemName: "magnifyingglass")
+        }
+    }
+}
+
+#if DEBUG
 public struct AboutDroidKaigiScreen_Previews: PreviewProvider {
     public static var previews: some View {
         Group {
-            AboutDroidKaigiScreen()
-                .previewDevice(.init(rawValue: "iPhone 12"))
-                .environment(\.colorScheme, .dark)
-            AboutDroidKaigiScreen()
-                .previewDevice(.init(rawValue: "iPhone 12"))
-                .environment(\.colorScheme, .light)
+            AboutDroidKaigiScreen(
+                store: .init(
+                    initialState: .init(),
+                    reducer: .empty,
+                    environment: {}
+                )
+            )
+            .previewDevice(.init(rawValue: "iPhone 12"))
+            .environment(\.colorScheme, .dark)
+
+            AboutDroidKaigiScreen(
+                store: .init(
+                    initialState: .init(),
+                    reducer: .empty,
+                    environment: {}
+                )
+            )
+            .previewDevice(.init(rawValue: "iPhone 12"))
+            .environment(\.colorScheme, .light)
```

Comment:
It seems better to use ForEach for color scheme.

Reply:
You mean like this ?
```Swift
  ForEach(ColorScheme.allCases, id: \.self) { colorScheme in
      AboutDroidKaigiScreen(
          store: .init(
              initialState: .init(),
              reducer: .empty,
              environment: {}
          )
      )
      .previewDevice(.init(rawValue: "iPhone 12"))
      .environment(\.colorScheme, colorScheme)
  }
```

Reply:
Yup. Exactly.